### DISCUSSION TITLE
How can I get my keyboard function keys to work within a Dos program?

### DISCUSSION THREAD

**Discussion by MTOLincoln** [joncampbell123/dosbox-x#4102] [Q&A]
For the life of me, I can't figure out a way to let my keyboard function keys pass through into my Dos program. My Dos program, for example, uses Shift-F1 to invoke help within the program, and Shift-F7 to start a Search, and a number of other similar configurations as well.

In an older version of DosBox, I was able to set a toggle to pass function keys through to the program or not. And that worked fine for my purposes. But can't figure how to do that with DosBox-X. Even using the drop-down menu's key-mapper. ???

I may be missing something obvious (?), but maybe obvious to others but not to me. If I'm being dumb, sorry.:(

Mike     

Some basic info: Trying to use dosbox-x on latest version of MacOS.

**Comment by Torinde** [comment]
Discussions tab is deprecated - please post your question in Issues. @aybe 

Try both SDL1 and SDL2 versions (they handle input differently)

**Comment by aybe** [comment]
https://github.com/joncampbell123/dosbox-x/issues/4150
